agent/bench-scripts/test-bin/pbench-start-tools

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
-
-echo "${0} ${*}" >> ${_testlog}
-
-PROG="$(basename "${0}")"
-
-opts=$(getopt -q -o d:g: --longoptions "dir:,group:" -n "mock-${PROG}" -- "${@}")
-if [[ ${?} -ne 0 ]]; then
-	printf -- "mock-%s: unrecognized argument, \"%s\"\n" "${PROG}" "${*}" >> ${_testlog}
-	exit 1
-fi
-eval set -- "${opts}"
-while true; do
-	opt="${1}"
-	shift
-	case "${opt}" in
-	-g|--group)
-		if [[ -n "${1}" ]]; then
-			group="${1}"
-			shift
-		fi
-		;;
-	-d|--dir)
-		if [[ -n "${1}" ]]; then
-			dir="${1}"
-			shift
-		fi
-		;;
-	--)
-		break
-		;;
-	*)
-		printf -- "mock-%s: unrecognized argument, \"%s\"\n" "${PROG}" "${opt}" >> ${_testlog}
```

Comment:
It would be good if the output from line 33 were distinguishable from the output from line 9.

_[I'm lodging this against the old code because it's the same as the new code, and because GitHub apparently won't let me comment on the new code!  (See Ticket #[1596679](https://support.github.com/ticket/1596679).]_